src/test/java/org/prgrms/cream/domain/user/controller/UserControllerTest.java

```diff
@@ -0,0 +1,397 @@
+package org.prgrms.cream.domain.user.controller;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import javax.validation.ConstraintViolationException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.prgrms.cream.domain.user.config.TestConfig;
+import org.prgrms.cream.domain.user.dto.UserSignUpTestRequest;
+import org.prgrms.cream.domain.user.dto.UserUpdateTestRequest;
+import org.prgrms.cream.domain.user.exception.DuplicateUserException;
+import org.prgrms.cream.domain.user.exception.InvalidArgumentException;
+import org.prgrms.cream.domain.user.exception.NotFoundUserException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.transaction.annotation.Transactional;
+
+@ActiveProfiles("local")
+@Transactional
+@AutoConfigureMockMvc
+@TestConfig
+@SpringBootTest
+class UserControllerTest {
+
+	private static UserSignUpTestRequest userSignUpRequest;
+	private static UserUpdateTestRequest userUpdateTestRequest;
+
+	@Autowired
+	private MockMvc mockMvc;
+	@Autowired
+	private ObjectMapper objectMapper;
+
+	@DisplayName("회원생성_테스트")
```

Comment:
```suggestion
	@DisplayName("회원생성 테스트")
```

여기서 언더바를 추가 안하고 편하게 작성하셔도 될거에요!

Reply:
아하!! 저는 언더바로 이어줘야하는 줄 알았습니당!! 감사합니다 ㅎㅎ